MiniTwitch.Helix/Responses/ConduitTransport.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace MiniTwitch.Helix.Responses;
+
+[JsonConverter(typeof(ConduitTransportConverter))]
+public abstract record ConduitTransport()
+{
+    public string Method { get; }
+
+    public sealed record WebSocket(string SessionId, DateTime ConnectedAt, DateTime DisconnectedAt) : ConduitTransport();
+    public sealed record Webhook(string Callback) : ConduitTransport();
+}
+
+public class ConduitTransportConverter : JsonConverter<ConduitTransport>
```

Comment:
Move this to Internal/Json/Converters.cs (or its own file)

Reply:
Fixed in 2e4a6b9ae27f337aade7ceeb56fddef351ab1c5b